types/nightwatch/index.d.ts

```diff
@@ -19,6 +19,18 @@ import { WebElement, WebElementPromise, By } from 'selenium-webdriver';
 
 export * from './globals';
 
+export const ELEMENT_KEY = 'element-6066-11e4-a52e-4f735466cecf';
+
+export interface JSON_WEB_OBJECT {
+    ELEMENT_KEY: string;
+    getID: () => string;
+}
+
+export type definition = string | ElementProperties | Element;
```

Comment:
this looks like overlook, should it follow JS convention for interface/types CamelCase notation?
```ts
export type Definition = string | ElementProperties | Element;
```

Reply:
Thanks for the review, I will update it.